File: 프로그래머스/1/12922. 수박수박수박수박수박수？/수박수박수박수박수박수？.js
function solution(n) {
    var answer = '';
    let i = 0;
    while (i < n) {
        i += 1
        if (i % 2 === 1) {
            answer += '수';
        } else if (i % 2 === 0) {
            answer += '박';
        }
    }
    return answer;
}

File: 프로그래머스/1/12922. 수박수박수박수박수박수？/수박수박수박수박수박수？.py
def solution(n):
    answer = ''
    i = 0
    while True:
        i += 1
        if i % 2 == 1:
            answer += '수'
        elif i % 2 == 0:
            answer += '박'
        
        if i == n:
            break
    return answer

print(solution(3))